pom.xml

```diff
@@ -166,6 +171,56 @@
     <junit.version>5.9.0</junit.version>
   </properties>
 
+  <profiles>
+    <profile>
+      <id>java11</id>
+      <activation>
+        <jdk>[11,)</jdk>
+      </activation>            
+      <properties>
+        <jboss.logging.version>3.5.3.Final</jboss.logging.version>
+        <org.bouncycastle.version>1.70</org.bouncycastle.version>
+      </properties>
+      <dependencies>
+        <dependency>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcpkix-jdk15on</artifactId>
+		  <version>${org.bouncycastle.version}</version>
+		  <scope>test</scope>
+		</dependency>
+	  </dependencies>
+	  <build>
+		<plugins>
+		  <plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-surefire-plugin</artifactId>
+			<configuration>
+			  <argLine> --add-opens java.base/jdk.internal.misc=ALL-UNNAMED -Dio.netty.tryReflectionSetAccessible=true</argLine>
+			</configuration>
+		  </plugin>
+		  <plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-enforcer-plugin</artifactId>
+			<executions>
+			  <execution>
+				<id>enforce-java</id>
+				<goals>
+				  <goal>enforce</goal>
+				</goals>
+				<configuration>
+				  <rules>
+					<requireJavaVersion>
+					  <version>[11,)</version>
+					</requireJavaVersion>
+				  </rules>    
+				</configuration>
+			  </execution>
+			</executions>
+		  </plugin>
```

Comment:
With any recent version of `jboss-parent` (since `33` if not before), you should be able to set the `jdk.min.version` property to `11` and this enforcement rule will be added automatically.